### PR TITLE
Separate config registers from the LLC implementation

### PR DESCRIPTION
**Pull request by creinwar**
Separate config registers allow use of custom or re-use of existing register files

### CODE REVIEW

**Comment by creinwar** [comment]
#2

File: Bender.yml [hunk 1]
```diff
@@ -12,8 +12,12 @@ dependencies:
   axi: { git: "https://github.com/pulp-platform/axi.git", version: 0.37.0 }
```
Comment (thommythomaso) [inline Bender.yml:12]: I will update AXI.

File: data/axi_llc_regs.hjson [hunk 2]
```diff
@@ -0,0 +1,158 @@
+// Copyright 2018-2021 ETH Zurich and University of Bologna.
+// Solderpad Hardware License, Version 0.51, see LICENSE for details.
+// SPDX-License-Identifier: SHL-0.51
+//
+// Authors: 
+// Nicole Narr <narrn@student.ethz.ch>
```
Comment (thommythomaso) [inline data/axi_llc_regs.hjson:6]: Lets use your ethz addresses here

**Review by thommythomaso — COMMENTED** [review]
Awesome work! Thank you so much.
I will fix some minor things myself, merge and release.

**Review by thommythomaso — DISMISSED** [review]

**Review by thommythomaso — DISMISSED** [review]

**Review by thommythomaso — APPROVED** [review]

**Pull request merged**